lib/addons/keda/index.ts

```diff
@@ -0,0 +1,143 @@
+import { Construct } from 'constructs';
+import { ManagedPolicy } from "aws-cdk-lib/aws-iam";
+import merge from "ts-deepmerge";
+import { HelmAddOn, HelmAddOnUserProps, HelmAddOnProps } from "../helm-addon";
+import { ClusterInfo, Values } from "../../spi";
+import { setPath, createNamespace } from "../../utils";
+
+/**
+ * User provided options for the Helm Chart
+ */
+export interface KedaAddOnProps extends HelmAddOnUserProps {
```

Comment:
minor: CRLF after line 11 and each field, e.g. lines 11, 15, 19. etc. 
I will add this to the lint check.

Reply:
For some reason, I cannot see it in VSCode, how ever i have changed it from LF to CRLF. It should be fixed now